### DISCUSSION TITLE
projectile nospread

### DISCUSSION THREAD

**Discussion by canedoly** [Fedoraware/Fedoraware#979] [Ideas]
apparently, you can do projectile nospread by setting the random seed/command number to a specific value and fixing the spread for that specific cmd num/random seed (beggar's bazooka and probably the rest)

**Comment by ghost** [comment]
it broke doubletap and lnx gave up.

**Comment by vaitomanocucpp** [comment]
imagine

**Comment by Eniac763** [comment]
> it broke doubletap and lnx gave up.

nooooooooooo

**Comment by ghost** [comment]
completedaged
